recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "24.2" %}
+{% set version = "24.3.1" %}
```

Comment:
There has been released a new 25.0 version https://pip.pypa.io/en/stable/news/#v25-0. I don't see breaking changes. Could you update to the latest?